x-pack/plugins/security_solution/public/management/cypress/.eslintrc.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Will this config still pull in the parent jslint configuration? I think it does because I don't see `root: true`, but just want  to confirm

Reply:
looks like it's working, at least it gives the usual 'missing header' in empty files